src/components/Button/Button.stories.tsx

```diff
@@ -22,3 +23,11 @@ export const ClickMe = Template.bind({});
 ClickMe.args = {
   label: 'Click me!',
 };
+
+// Example for @storybook/addon-links: link from one component to another
+export const first = () => (
+  <button onClick={linkTo('Components/Button', 'second')}>Go to Second</button>
+);
+export const second = () => (
+  <button onClick={linkTo('Components/Button', 'first')}>Go to First</button>
+);
```

Comment:
Example for addon-links. Better example would be linking e.g. Login button to Login form.